src/OSPSuite.Presentation/Presenters/ObservedData/DataRepositoryDataPresenter.cs

```diff
@@ -90,9 +90,7 @@ public void AddData(int rowIndex)
       /// <returns></returns>
       public IEnumerable<string> GetCellValidationErrorMessages(int rowIndex, int columnIndex, string newValue)
       {
-         if (!float.TryParse(newValue, out var proposedValue))
```

Comment:
This is not required because the method throws an exception of the value is not a float. 
Moreover, this cannot happen because we have a formatting in place